R/get_stan_data.R

```diff
@@ -239,9 +239,9 @@ get_stan_data <- function(input_count_data,
   # Get the total pop, coming from the larger population generating the
   # count data
   pop <- input_count_data |>
-    dplyr::select(total_pop) |>
+    dplyr::select("total_pop") |>
```

Comment:
This seems like a big change; am I missing something? cc @kaitejohnson

Reply:
No so basically I just want the unique values of the total population column in the input count data here. 

I tend to do this with `select` for the column name and then `pull()` to get the vector, and then `unique()`. I often forget that this can be done more succinctly with `distinct(col_name)` and `pull()` bc distinct automatically does the selecting.